.pre-commit-config.yaml

```diff
@@ -33,8 +33,3 @@ repos:
     rev: 4.0.1
     hooks:
       - id: flake8
-
```

Comment:
Same as in https://github.com/stackabletech/trino-operator/pull/212: Intentionally removed? (If yes maybe operator-rs would be a great place :))

Reply:
Yeah we remove it from templating for now.